assets/kube-controller-manager/config.yaml

```diff
@@ -40,8 +40,12 @@ extendedArguments:
   - configmaps
   leader-elect:
   - 'true'
+  leader-elect-lease-duration:
+  - 60s
+  leader-elect-renew-deadline:
+  - 15s
   leader-elect-retry-period:
-  - 3s
+  - 10s
```

Comment:
It think this is too high and yields zero seconds of Kubernetes API server downtime per https://github.com/openshift/library-go/blob/4b9033d00d37b88393f837a88ff541a56fd13621/pkg/config/leaderelection/leaderelection.go#L90-L95.  It think we should probably use `3s` here.  Then maybe we can bump further in a OpenShift 4.9.

Reply:
retry-period updated to 3s